web/impact/impact/tests/test_graphql.py

```diff
@@ -604,23 +604,6 @@ def test_query_for_user_without_confirmed_mentor_program_families(self):
             self.assertEqual(expert_profile["confirmedMentorProgramFamilies"],
                              [])
 
-    def test_user_cannot_view_profile_with_non_current_allowed_roles(self):
```

Comment:
Rather than deleting this test, let's change it to reflect the behavior change

Reply:
Added test